openstudiocore/src/model/ThermalZone.hpp

```diff
@@ -416,24 +416,47 @@ class MODEL_API ThermalZone : public HVACComponent {
    */
   boost::optional<double> sequentialCoolingFraction(const ModelObject& equipment) const;
 
+  /** Return the Sequential Cooling Fraction Schedule of equipment.
+   *  Returns nothing if when equipment is not in the ZoneHVACEquipmentList, its heating priority is zero,
+   *  or the loadDistributionScheme isn't 'Sequential'
+   */
+  boost::optional<Schedule> sequentialCoolingFractionSchedule(const ModelObject& equipment) const;
+
   /** Return the Sequential Heating Fraction of equipment.
    *  Returns nothing if when equipment is not in the ZoneHVACEquipmentList, its cooling priority is zero,
    *  or the loadDistributionScheme isn't 'Sequential'
    */
   boost::optional<double> sequentialHeatingFraction(const ModelObject& equipment) const;
 
+  /** Return the Sequential Heating Fraction Schedule of equipment.
+   *  Returns nothing if when equipment is not in the ZoneHVACEquipmentList, its cooling priority is zero,
+   *  or the loadDistributionScheme isn't 'Sequential'
+   */
+  boost::optional<Schedule> sequentialHeatingFractionSchedule(const ModelObject& equipment) const;
+
   /** Set the Sequential Cooling Fraction of equipment.
    *  Returns false when equipment is not in the ZoneHVACEquipmentList, its cooling priority is zero,
    *  or the loadDistributionScheme isn't 'Sequential'
    */
   bool setSequentialCoolingFraction(const ModelObject& equipment, double fraction);
 
+  /** Set the Sequential Cooling Fraction Schedule of equipment.
+   *  Returns false when equipement is not in the ZoneHVACEquipmentList, its cooling priority is zero,
+   *  or the loadDistributionScheme isn't 'Sequential'
+   */
+  bool setSequentialCoolingFraction(const ModelObject& equipment, const Schedule& schedule);
+
   /** Set the Sequential Heating Fraction of equipment.
    *  Returns false when equipment is not in the ZoneHVACEquipmentList, its heating priority is zero,
    *  or the loadDistributionScheme isn't 'Sequential'
    */
   bool setSequentialHeatingFraction(const ModelObject& equipment, double fraction);
```

Comment:
Mark as OS_DEPRECATED

Reply:
Hum, if someone were to advocate that this setter is useful and shouldn't be marked as deprecated, I wouldn't fight back too hard.

Reply:
We don't want to deprecate these, @joseph-robertson will review and provide another PR if needed